packages/contracts/test/plugins/governance/majority-voting/token/token-voting.ts

```diff
@@ -295,78 +319,126 @@ describe('TokenVoting', function () {
       await setTotalSupply(1);
     });
 
-    it('reverts if the user is not allowed to create a proposal', async () => {
-      votingSettings.minProposerVotingPower = 1;
+    context('minProposerVotingPower == 0', async () => {
+      beforeEach(async () => {
+        votingSettings.minProposerVotingPower = 0;
+        await voting.initialize(
+          dao.address,
+          votingSettings,
+          governanceErc20Mock.address
+        );
+      });
 
-      await voting.initialize(
-        dao.address,
-        votingSettings,
-        governanceErc20Mock.address
-      );
+      it('creates a proposal if `_msgSender` owns no tokens and has not tokens delegated to her/him in the current block', async () => {
+        await setBalances([
+          {
+            receiver: signers[1].address,
+            amount: votingSettings.minProposerVotingPower,
```

Comment:
Generated confusion otherwise

```suggestion
            amount: votingSettings.minProposerVotingPower, // equals 0
```